aws/provider.go

```diff
@@ -148,6 +148,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_api_gateway_api_key":                dataSourceAwsApiGatewayApiKey(),
 			"aws_api_gateway_resource":               dataSourceAwsApiGatewayResource(),
 			"aws_api_gateway_rest_api":               dataSourceAwsApiGatewayRestApi(),
+			"aws_api_gateway_stage":                  dataSourceAwsApiGatewayStage(),
```

Comment:
This new data source is missing the following documentation:
* A sidebar link in `website/aws.erb`
* A new data source documentation page: `website/docs/d/api_gateway_stage.html.markdown`

Reply:
Just added!